lib/index.js

```diff
@@ -113,4 +143,70 @@ Drip.prototype.normalize = function(obj){
   }
 
   return ret;
-}
+};
+
+/**
+ * Limits the integration to send too many request to the partner.
+ *
+ * @param {Function} headers
+ * @param {Function} completion
+ * @api private
+ */
+
+Drip.prototype.setLimit = function(headers, completion) {
+  var appId = this.settings.account;
+  var key = ['drip', appId].join(':');
+  var redis = this.redis();
+  var self = this;
+
+  var limit = {
+    total: headers['x-ratelimit-limit'],
+    remain: headers['x-ratelimit-remaining'],
+    reset: Date.now() + (1000 * 60 * 3)
```

Comment:
They don't include it in the header, but experimentally, the limits reset at hour boundaries. We could set the reset time here a little more accurately (although then we run the risk of it being very wrong if they change their rate limiting implementation)